### PR TITLE
Describe pros/cons vs "single step" transition in "A Path to Proof-of-Stake Zcash"

### LINKED ISSUE #46: Document the Single-Step PoS transition alternative to Multi-Step PoS transition and our rationale.
### Suggested Improvement

Add a section to the book documenting the alternative of transitioning the Zcash network to PoS in "a single step" rather than having an intermediate phase, such as the TFL approach.

We currently intend to continue pursuing the TFL approach for now, so the book needs to provide the rationale.

### PR DESCRIPTION
**Pull request by nathan-at-least**
**Note:** This PR extends #106 and should be reviewed/merged after #106.

This also removes the "goals" section of that chapter, since it's redundant with `./overview/design-goals.md`. Instead we link to that.

This PR closes #46 .

### CODE REVIEW

File: src/introduction/a-path-to-pos-zcash.md [hunk 1]
```diff
@@ -14,35 +14,58 @@ We envision a *proof-of-stake transition path* as one of the potential paths wit
...
 With this approach, the Zcash Tech Tree with the [TFL](../terminology.md#definition-tfl) approach might look something like this:
 
 ```dot process
 {{#include ../diagrams/zcash-tech-tree-tfl-steps.dot}}
 ```
 
-## Design Goals for a Hybrid PoW/PoS System
+## Why Two Steps?
+
+One question we've gotten in proposing this approach is why take a two step process with an intermediate [hybrid consensus](../terminology.md#definition-hybrid-consensus) protocol, rather than a single transition directly to a [PoS](../terminology.md#definition-pos) protocol?
```
Comment (daira) [inline src/introduction/a-path-to-pos-zcash.md:36]:
```suggestion
One question we've gotten in proposing this approach is why take a two-step process with an intermediate [hybrid consensus](../terminology.md#definition-hybrid-consensus) protocol, rather than a single transition directly to a [PoS](../terminology.md#definition-pos) protocol? 
```

File: src/introduction/a-path-to-pos-zcash.md [hunk 2]
```diff
@@ -14,35 +14,58 @@ We envision a *proof-of-stake transition path* as one of the potential paths wit
...
+
+#### Cons
+- If there is any unforeseen show-stopping problem in the new protocol or the transition process, we’d have to react to a network-wide issue.
+- It may be more likely to cause ecosystem disruption; unforeseen differences between PoW and PoS might cause various kinds of snags or papercuts throughout the ecosystem, and these would all pop up around the same time, which may lead to a loss of confidence/retention/adoption or at the very least inconvenience many users for some time.
+- Losing miners: since the transition would be all at once, we may lose some number of miners, who are participants and users in the ecosystem. Miners may leave prior to the transition in order to take care of their own needs. If there is some show-stopper in the transition, one possible short-term mitigation would be to fall back on PoW which is well-known, but if we’ve lost most miners, that may no longer be viable.
```
Comment (daira) [inline src/introduction/a-path-to-pos-zcash.md:51]:
```suggestion
- Losing miners: since the transition would be all at once, we may lose some number of miners, who are participants and users in the ecosystem. Miners may leave prior to the transition in order to take care of their own needs. If there is some show-stopper in the transition, one possible short-term mitigation would be to fall back on PoW which is well known, but if we’ve lost most miners, that may no longer be viable.
```

File: src/introduction/a-path-to-pos-zcash.md [hunk 3]
```diff
@@ -14,35 +14,58 @@ We envision a *proof-of-stake transition path* as one of the potential paths wit
...
+- Retaining miners during a hybrid phase: while it is true that a hybrid protocol will lower miner revenue (since we aim to maintain the issuance schedule constraints), there is also more possibility and likelihood of keeping some of these users engaged. For example, they may begin participating in staking services (either as delegators or as infrastructure operators). If that is successful, then they’re also more likely to remain engaged in the subsequent transition to pure PoS.
+- This general approach was demonstrated successfully by Ethereum, which is the largest or second largest cryptocurrency network for several important metrics (e.g. market cap, fees paid, user and developer activity, …). So we know this can be done well without disruptions.
 
-With the motivation and goals for a first-step hybrid PoW/PoS protocol in mind, we introduce the specifics of TFL in the next subchapter.
+#### Cons
+- The intermediate hybrid step will be a more novel and less well understood protocol. (It will necessarily be fairly different from Ethereum’s Beacon chain era.)
```
Comment (daira) [inline src/introduction/a-path-to-pos-zcash.md:64]: Why will it necessarily be fairly different from Ethereum's Beacon chain era? Despite the design work on Crosslink which is indeed different, I think we still haven't excluded the option to "just replicate as closely as possible what Ethereum did".

**Review by daira — APPROVED** [review]
ACK with comments.

File: src/introduction/a-path-to-pos-zcash.md (hunk 3 above)
Comment (nathan-at-least) [reply src/introduction/a-path-to-pos-zcash.md:64]: There are many many details for Beacon Chain that would be Ethereum specific. Some user-facing examples:

- I don't know if the beacon chain provided finality!
- Users could deposit to the beacon chain for staking prior to the beacon chain launching (I believe).
- Users could stake and run beacon chain validators without any ability to withdraw to L1 for a long time (~18 months).
- Users could deposit to stake the Beacon chain by depositing into a specific contract with a well-known address, so the UX was specific to Ethereum's contract addresses.
- I don't know if non-beacon nodes had to refer to beacon blocks, as Crosslink does.
- The staking rates, slashing amounts, slashing triggers, etc… were all likely to be different from our design.
- I believe it may have activated based on amount of stake deposited, whereas we may want to just have a pre-defined activation height? (Open question.)

So basically I think there are many unknowns (for me) and I simply don't want to lead any readers into _expecting_ any feature/behavior parity with the Beacon Chain approach.

We can always explicitly decide we want to adopt similar design goals! I just don't want to pre-commit to parity.

(BTW, one aspect I really liked is the despits-before-activate because that serves as an unambiguous skin-in-the-game permissionless signal of confidence in the new protocol feature.)

**Comment by nathan-at-least** [comment]
I'm merging despite one outstanding comment on the theory we can address that concern in future iterations if needed.
